devicetypes/smartthings/osram-lightify-gardenspot-mini-rgb.src/osram-lightify-gardenspot-mini-rgb.groovy

```diff
@@ -21,9 +21,6 @@ metadata {
         attribute "colorName", "string"
 
         command "setAdjustedColor"

```

Comment:
I couldn't see any reason that we need `setAdjustedColor` anymore, but it should be looked at carefully during QA.